src/ResourceManagement/gulpfile.js

```diff
@@ -6,7 +6,7 @@ var spawn = require('child_process').spawn;
 var fs = require('fs');
 
 const mappings = require('./api-specs.json');
-const defaultSpecRoot = "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/current";
+const defaultSpecRoot = "https://raw.githubusercontent.com/Azure/azure-rest-api-specs/master";
 
```

Comment:
why do we move from current to master?

Reply:
azure-rest-api-specs people decided to delete the "current" branch today and put all the recent stuff into master.

Reply:
@RikkiGibson then please also do open the similar PR in the azure-sdk-for-java repo. Thanks!